<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>545</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="verticalGroupBox">
         <property name="font">
          <font>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(245, 245, 245);</string>
         </property>
         <property name="title">
          <string>File Settings</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPushButton" name="pb_folderPath">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>40</width>
                <height>40</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>16</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Select Folder</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="le_path">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <pointsize>16</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Data Name :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="le_name">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="font">
               <font>
                <pointsize>16</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Extentsion:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cmb_ext">
              <property name="font">
               <font>
                <pointsize>16</pointsize>
               </font>
              </property>
              <item>
               <property name="text">
                <string>.czi</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>.tif</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QGroupBox" name="horizontalGroupBox">
           <property name="font">
            <font>
             <pointsize>24</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(245, 245, 245);</string>
           </property>
           <property name="title">
            <string>Motor Settings</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QLabel" name="label_7">
                <property name="font">
                 <font>
                  <pointsize>16</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Step Size(Î¸):</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="cmb_step">
                <property name="minimumSize">
                 <size>
                  <width>55</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>40</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>16</pointsize>
                 </font>
                </property>
                <property name="currentIndex">
                 <number>2</number>
                </property>
                <item>
                 <property name="text">
                  <string>2</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>5</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>10</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>20</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>45</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="font">
                 <font>
                  <pointsize>16</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Acceleration(%)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="sb_acc">
                <property name="minimumSize">
                 <size>
                  <width>55</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>40</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>16</pointsize>
                 </font>
                </property>
                <property name="maximum">
                 <number>100</number>
                </property>
                <property name="value">
                 <number>100</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="font">
                 <font>
                  <pointsize>16</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Speed(%)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="sb_speed">
                <property name="minimumSize">
                 <size>
                  <width>55</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>40</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>16</pointsize>
                 </font>
                </property>
                <property name="maximum">
                 <number>100</number>
                </property>
                <property name="value">
                 <number>100</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="font">
          <font>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="title">
          <string>Acquisition Settings</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="font">
               <font>
                <pointsize>16</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Experiment:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="le_experiment">
              <property name="font">
               <font>
                <pointsize>16</pointsize>
               </font>
              </property>
              <property name="text">
               <string>testBenjamin</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="font">
               <font>
                <pointsize>16</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Number of image:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="sb_image">
              <property name="minimumSize">
               <size>
                <width>55</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>16</pointsize>
               </font>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="value">
               <number>18</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pb_start">
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Start Acquisition</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
